File: python/service/logs.py
from config.env import Env
from library.file_eo import FileEo

# 日志
class Logs:

  # 写入文件
  def File(file: str, content: str):
    FileEo.Root = Env.root_dir
    FileEo.WriterEnd(file, content+"\n")

File: php/service/Logs.php
<?php
namespace Service;

use Library\FileEo;

/* 日志 */
class Logs extends Base {

  /* 写入文件 */
  static function File(string $file='', $content=''){
    FileEo::WriterEnd($file, json_encode($content)."\n");
  }

}